storage/memory/log_storage.go

```diff
@@ -427,11 +427,10 @@ func (t *readOnlyLogTX) GetUnsequencedCounts(ctx context.Context) (storage.Count
 	ret := make(map[int64]int64)
 	for id, tree := range t.ms.trees {
 		tree.RLock()
-		defer tree.RUnlock() // OK to hold until method returns.
-
 		k := unseqKey(id)
 		queue := tree.store.Get(k).(*kv).v.(*list.List)
 		ret[id] = int64(queue.Len())
+		tree.RUnlock() // OK to hold until method returns.
```

Comment:
Probably should remove the comment

Reply:
Good point, I'll fix that. The decrease in code coverage showed there are no tests for `RunInAdminSnapshot` so I'm going to fix that as well.

Reply:
Still got the out-of-date comment.

Reply:
Done.